docs/integrate/etl.md

```diff
@@ -104,6 +104,32 @@ A few companies are specializing in offering managed Flink services.
 :::
 
 
+## Apache Hop
+
+```{div}
+:style: "float: right"
+[![](https://hop.apache.org/img/hop-logo.svg){w=180px}](https://hop.apache.org/)
+```
+
+[Apache Hop] aims to be a modern, open source data integration platform that is
+easy to use, fast and flexible. It facilitates all aspects of data and metadata
+orchestration.
+
+- **Visual development** enables developers to be more productive than they can
+  be through code.
+
+- **Workflows and pipelines** can be designed in the Hop Gui and run on the Hop
+  native engine (local or remote), or on Spark, Flink, Google Dataflow or AWS EMR
+  through Beam. _Design once, run anywhere._
+
+- **Lifecycle Management** enables developers and administrators to switch between
+  projects, environments and purposes without leaving your train of thought.
+
+```{seealso}
+[CrateDB and Apache Hop]
+```
```

Comment:
There are no references to relevant tutorials yet. Just let us know if you intend to publish a corresponding article on the community forum, whether you would want us to do it, extended by exercising a relevant snippet on behalf of [cratedb-examples](https://github.com/crate/cratedb-examples), or if you can do that.

A very little walkthrough in form of a README.md, added on behalf of a patch and PR, will be sufficient to enable others to continue working from there.

In any case, it is not a blocker for this to be merged.

/cc @matkuliak

Reply:
We are reviewing a draft for a tutorial, please wait to merge this until the tutorial is published as there are some details to consider to make it work with CrateDB

Reply:
Apologies for the delay, the tutorial is out now https://community.cratedb.com/t/using-apache-hop-with-cratedb/1754

Reply:
Yes, but could we link the tutorial? because it does not work out-of-the-box without the workaround for `pg_user`

Reply:
Sure. The tutorial is linked into the CrateDB Guide.

-- https://github.com/crate/cratedb-guide/pull/83